### PR TITLE
chore(react-component): low lever ItemTitle component

### LINKED ISSUE #61: Title React component

### PR DESCRIPTION
**Pull request by sofiapoh**
closes #61

### CODE REVIEW

File: src/components/app.jsx [hunk 1]
```diff
@@ -1,5 +1,14 @@
+/* eslint-disable react/prefer-stateless-function */
 import React from 'react';
 
-const App = () => (<h1>Hello world!!!</h1>);
+import ItemTitle from './ItemTitle.jsx';
```
Comment (matthewiiv) [inline src/components/app.jsx]: We don't want to import this here on the master version. I suggest keeping it on your branch while you're testing but we should keep the master branch clear of code we know we are going to remove.

**Comment by matthewiiv** [comment]
File names should be lower case with underscores. So item_title.jsx

**Comment by daymos** [comment]
type in commit should be feat instead of chore

**Comment by sofiapoh** [comment]
intese! okay, fixing!

**Pull request merged**